src/pkg/scan/dao/scanner/model.go

```diff
@@ -105,10 +106,11 @@ func (r *Registration) Validate(checkUUID bool) error {
 		return errors.New("missing registration name")
 	}
 
-	err := checkURL(r.URL)
+	url, err := lib.ValidateHTTPURL(r.URL)
```

Comment:
if the unexported func `checkURL` still needed?

Reply:
ok, let me check. If no use, I'll remove it.